fr.hjson

```diff
@@ -308,66 +308,66 @@
 
   "joinconfig.chatname.help": "<code>/%s</code><b> vous permet de définir le nom du chat. Ce nom apparaîtra toujours devant tout titre de chat défini à l'aide de </b><code>/t</code> <b>ou</b> <code>/title</code><b>.\n\nVotre message devrait ressembler à ceci:</b>\n<code>/%s MyChatName</code>"
-  "joinconfig.chatname.replyToTextHint": "<b>Si vous répondez à un autre message pour définir le nom du chat, il doit être en texte uniquement.</b>"
-  "joinconfig.chatname.success": "<b>Votre nom de chat a été défini sur:</b>\n%s\n\nCeci apparaîtra au début de chaque jeu de titres de chat en utilisant <code>/t</code> ou <code>/title</code>."
+  "joinconfig.chatname.replyToTextHint": "<b>Si vous répondez à un autre message pour définir le nom du chat, il doit être composé de texte uniquement.</b>"
+  "joinconfig.chatname.success": "<b>Votre nom de chat a été défini comme étant:</b>\n%s\n\nCeci apparaîtra au début de chaque titre de chat défini à l'aide de <code>/t</code> ou <code>/title</code>."
 
 
   /* ---------------------------------------------------------------------
                         JoinVerify - Used for /JoinVerify
      --------------------------------------------------------------------- */
   # ⚠️ IMPORTANT
   "joinverify.callbacks": {
-    "correctAnswerText": "Correct! %s plus à parcourir..."
-    "incorrectAnswerText": "Faux. 3 de plus à parcourir..."
-    "success": "<b>Succès!\n\nBon travail humain, tu peux maintenant chatter: </b><code>%s</code>"
-    "successAnswerText": "Succès! Vous pouvez chatter %s à présent..."
-    "manuallyTrusted": "%s<b> a fait confiance manuellement </b>%s<b>.</b>"
-    "manuallyTrustedAnswerText": "Utilisateur de confiance"
-    "expired": "<b>Cette demande de vérification a expiré.</b>\n\nVous <b>peut avoir été manuellement banni ou approuvé par les administrateurs du groupe.</b>Ou, c'est une demande très ancienne."
-    "expiredAnswerText": "Vérifier la demande expirée"
-
-    "settingsUnavailable": "<b>Join Verify n'est pas activé pour le moment, activez-le d'abord pour modifier les paramètres:</b>"
-    "setTimeLimit": "<b>Choisissez une limite de temps pour rejoindre Vérifier</b>\n\nCombien de minutes doivent être accordés aux nouveaux membres avant d'être bannis?"
+    "correctAnswerText": "Correct! Encore %s ..."
+    "incorrectAnswerText": "Faux. Encore 3..."
+    "success": "<b>Succès !\n\nBon travail humain, vous pouvez maintenant discuter dans le groupe: </b><code>%s</code>"
+    "successAnswerText": "Succès ! vous pouvez maintenant discuter dans le groupe %s ..."
+    "manuallyTrusted": "%s<b> a approuvé manuellement à </b>%s<b>.</b>"
+    "manuallyTrustedAnswerText": "Utilisateur approuvé"
+    "expired": "<b>Cette demande de vérification a expiré.\n\n Il se peur que </b>vous <b>ayez été manuellement banni ou approuvé par les administrateurs du groupe.</b>Ou, c'est une très vieille demande ."

```

Comment:
Since the newline is here, we can get rid of the space

```suggestion
    "expired": "<b>Cette demande de vérification a expiré.\n\nIl se peur que </b>vous <b>ayez été manuellement banni ou approuvé par les administrateurs du groupe.</b>Ou, c'est une très vieille demande ."
```